tests/e2e/steps.py

```diff
@@ -0,0 +1,14 @@
+from testflows.core import *
+from testflows.connect import Shell
+
+
+@TestStep(Given)
+def get_shell(self, timeout=300):
+    """Create shell terminal."""
+    try:
+        shell = Shell()
+        shell.timeout = timeout
+        yield shell
```

Comment:
are you sure finally block will executed properly ?

Reply:
Yes, it is common syntax, it calls final block in the end of scenario.